packages/features/ee/teams/lib/payments.ts

```diff
@@ -19,6 +19,9 @@ const teamPaymentMetadataSchema = z.object({
   orgSeats: teamMetadataSchema.unwrap().shape.orgSeats,
 });
 
+const DEFAULT_SELF_SERVE_ORG_SEATS = MINIMUM_NUMBER_OF_ORG_SEATS; // Minimum number of seats for an organization 30
+const DEFAULT_SELF_SERVE_ORG_PRICE_PER_SEAT = 3700; // $37.00 per seat
```

Comment:
Should we not link this something in stripe? 

If not, maybe at least making it an env var?

wouldn’t want to have to change the code to change pricing.

Reply:
I will make these env vars for now - i think reaching out to stripe for this might be a bit OTT for right now